Sources/BrowserServicesKit/Common/Extensions/StringExtension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  StringExtension.swift
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+extension String {
```

Comment:
I know we discussed we probably don’t want to have Common section in BSK, but I would end up having `func drop(prefix: String)` defined in three fileprivate extensions. 🤷‍♂️

Reply:
It makes sense. In my opinion we could even make those extensions public just to eliminate the code duplication.
I understand the wish not to put all the kinds of extensions used around the codebase into the Common Lib but I find nothing wrong in putting the truly commonly used extensions in here.
The only question I have about the `drop(prefix:)` function is it's breaking the Swift naming convention as it's returning a modified value and not mutating, `dropping(prefix:)` would fit better for it.